strr-host-pm-web/app/components/form/DefineYourRental/UnitRequirements/Error.vue

```diff
@@ -63,7 +63,7 @@ function handleContinueApp () {
                 >
                   <template #linkAllRules>
                     <a
-                      href=""
+                      href="https://www2.gov.bc.ca/gov/content/housing-tenancy/short-term-rentals"
```

Comment:
Can we put those as configs?
or in the i18n?

Reply:
yeah sure I can do that

Reply:
fixed